br/pkg/lightning/backend/external/byte_reader.go

```diff
@@ -17,33 +17,50 @@ package external
 import (
 	"context"
 	"io"
-	"sync/atomic"
 
-	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/br/pkg/membuf"
 	"github.com/pingcap/tidb/br/pkg/storage"
 	"github.com/pingcap/tidb/util/logutil"
 	"github.com/pingcap/tidb/util/mathutil"
 	"go.uber.org/zap"
 )
 
-// ConcurrentReaderBufferSize is the buffer size for concurrent reader.
-var ConcurrentReaderBufferSize = 4 * 1024 * 1024
+var (
+	// ConcurrentReaderBufferSize is the buffer size for concurrent reader.
+	ConcurrentReaderBufferSize = 4 * 1024 * 1024
```

Comment:
Is this mean reader buffer size 4M?

Reply:
it's per concurrency of a reader, I'll rename it.

Reply:
https://github.com/pingcap/tidb/pull/47239/commits/e8773f5304fd46e87e21a51ff8bacb28b440e4e5